### DISCUSSION TITLE
Swagger validation does not return json object with error message for wrong uuid filed in request body

### DISCUSSION THREAD

**Discussion by sebastianpawlaczyk** [oapi-codegen/oapi-codegen#1462] [Q&A]
Hi,

As in title: I have uuid filed in request body and using GetSwagger function it returns 400 code if I provide wrong uuid value, but I do not receive error message.

```
nodeUUID:
    type: string
    format: uuid
```

**Comment by jamietanna** [comment]
Hey, can you please provide a minimal example? Is this for a field in the request body, or a path parameter? Are you using the kin-openapi middleware with this?
